user_manual/pim/sync_ios.rst

```diff
@@ -5,11 +5,12 @@ Synchronizing with iOS
 Calendar
 --------
 
-#. Open the settings application.
-#. Select Passwords and Accounts.
+#. Open the Settings application.
+#. Select Calendar.
+#. Select Accounts.
 #. Select Add Account.
 #. Select Other as account type.
-#. Select Add CalDAV account.
+#. Under "CALENDARS", select Add CalDAV account.
```

Comment:
Wouldn't it make more sense to write:
`Under "CALENDARS", select "Add CalDAV account".`